docs/sources/reference/run.md

```diff
@@ -587,6 +588,21 @@ division of CPU shares:
     101    {C1}		1	100% of CPU1
     102    {C1}		2	100% of CPU2
 
+### CPU period constraint
+
+The default period of CPUs is 100ms. We can use `--cpu-period` to set the 
```

Comment:
I don't think `The default period of CPUs is 100ms.` is right - isn't it a CFS scheduler default period - used by Linux..

ie, its not a CPU feature, but a Linux scheduler feature.